AngularJS/src/services/sessionService.js

```diff
@@ -0,0 +1,21 @@
+angular.module('app-bootstrap').service('sessionService', [
+  'storageService', '$http',
+  function (storageService, $http) {
+    let login = !!storageService.getSessionToken();
+    if (login) {
+      $http.defaults.headers.common.authorization = storageService.getSessionToken();
+    }
+    this.setUserInfo = (userData) => {
+      login = true;
+      storageService.setSessionToken(userData.access_token);
+    };
+
+    this.deleteUserInfo = () => {
+      login = false;
+      storageService.removeSessionToken();
+    };
+    this.loginState = () => {
+      return login;
```

Comment:
If it's just this return, there's no need for the curly braces in the arrow function:

```js
this.loginState = () => login;
```

That will work just fine (the arrow function has an implicit return). BTW, if you are returning a variable, just set it as a controller variable.